phys2bids/interfaces/txt.py

```diff
@@ -77,3 +73,121 @@ def populate_phys_input(filename, chtrig):
     timeseries.pop(0)
     ordered_timeseries = ordered_timeseries + timeseries
     return BlueprintInput(ordered_timeseries, freq, names, units)
+
+
+def acq_read(channel_list, chtrig, header=[]):
+    """
+    Reading function for acq files in txt format
+    Parameters
+    ----------
+    channel_list: list
+        list with channels only
+    chtrig : int
+        index of trigger channel
+    header: list
+        list with that contains file header
+    Returns
+    -------
+    BlueprintInput
+    See Also
+    --------
+    physio_obj.BlueprintInput
+    """
+    # get frequency
+    if len(header) == 0:
+        raise AttributeError('Files without header are not supported yet')
+    interval = header[1][0].split()
+    if interval[-1].split('/')[0] not in ['hr', 'min', 'sec', 'µsec', 'msec']:
+        raise AttributeError(f'Interval unit "{interval[-1]}" is not in a valid LabChart'
```

Comment:
At the very least, the name of the software should change ;)
Can you put a reference for this point, if it exists?

Reply:
In the case of ``acq`` I couldn't find the proper units for interval, maybe people from Chicago can help?

Reply:
They don't use AcqKnowledge - maybe someone else can. For the moment, I would not raise an error in this case - but you can raise a warning!

Reply:
I don't think µsec will be an option... Specially because it's a special character.

Reply:
Actually, it's an option in labchart, why not in AcqKnowlegde?